app/templates/main/search-results-summary.html

```diff
@@ -11,7 +11,7 @@
             <div class="mobile__search__container-block"></div>
             <div class="govuk-grid-row">
                 <div class="govuk-grid-column-two-thirds">
-                    {% if filters["query"]|length >0 %}
+                    {% if filters["query"].strip() |length >0 %}
```

Comment:
I notice we call .strip alot, maybe we can just do it once at the top of the route?